File: relationship-manager/risk-profiling-service/src/main/java/com/bank/rm/risk/RiskProfilingApplication.java
package com.bank.rm.risk;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
@SpringBootApplication(scanBasePackages = {"com.bank.rm.risk", "com.bank.rm.common"})
public class RiskProfilingApplication {
    public static void main(String[] args) { SpringApplication.run(RiskProfilingApplication.class, args); }
}

File: relationship-manager/risk-profiling-service/src/main/kotlin/com/bank/rm/risk/RiskProfilingApplication.kt
package com.bank.rm.risk

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.bank.rm.risk", "com.bank.rm.common"])
class RiskProfilingApplication

fun main(args: Array<String>) {
    runApplication<RiskProfilingApplication>(*args)
}
